src/components/share-results.tsx

```diff
@@ -59,14 +66,18 @@ export const ShareResults: React.FC<Props> = (props: any) => {
 
   return (
     hasClasses && (
-      <ShareContainer>
-        <FacebookShareButton href={facebookHref}>
-          {t('share.facebookButton')}
-        </FacebookShareButton>
-        <TwitterSharebutton href={twitterHref}>
-          {t('share.twitterButton')}
-        </TwitterSharebutton>
-      </ShareContainer>
+      <div>
+        <CtaText>{t('share.CTA')}</CtaText>
```

Comment:
Looks odd that it's wrapped in another div, it should be within the `ShareContainer` imo if possible

Reply:
putting it in a ShareContainer will make the text be on the left side of the sharing buttons

Reply:
That's fine, I didn't get the desired layout. Had to do an additional little change, looks good now!